exasol_script_languages_container_ci_setup/health_check.py

```diff
@@ -27,64 +26,67 @@ def check_shell_cmd(cmd: str) -> bool:
     return result.returncode == 0
 
 
-def is_supported_platform(**kwargs) -> Optional[ErrorMessageBuilder]:
+def is_supported_platform(**kwargs) -> Optional[error._error.Error]:
     """
     Checks weather or not the current platform is supported.
     """
     if sys.platform not in SUPPORTED_PLATFORMS:
-        return (
-            ExaError.message_builder("E-SLCCS-02")
-            .message("The platform you are running on is not supported.")
-            .mitigation(
-                "Make sure you are using one of the following platforms: {SUPPORTED_PLATFORMS}."
-            )
+        return error.ExaError(
+            code="E-SLCCS-02",
+            message="The platform you are running on is not supported.",
+            mitigations=[
+                "Make sure you are using one of the following platforms: {{supported_platforms}}."
+            ],
+            parameters={"supported_platforms": str(SUPPORTED_PLATFORMS)},
         )
 
 
-def aws_cli_available(**kwargs) -> Optional[ErrorMessageBuilder]:
+def aws_cli_available(**kwargs) -> Optional[error._error.Error]:
     """Checks weather AWS cli is installed"""
     command = "aws --help"
     if not check_shell_cmd(command):
-        return (
-            ExaError.message_builder("E-SLCCS-03")
-            .message("AWS CLI not installed.")
-            .mitigation(
-                "Install AWS CLI. "
-                "Goto https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html"
-            )
+        return error.ExaError(
+            code="E-SLCCS-03",
+            message="AWS CLI not installed.",
+            mitigations=[
+                "Install AWS CLI. Goto https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html"
+            ],
+            parameters={},
         )
 
 
-def aws_profile_valid(aws_profile: str) -> Optional[ErrorMessageBuilder]:
+def aws_profile_valid(aws_profile: str) -> Optional[error._error.Error]:
```

Comment:
That looks wrong

Reply:
This is the class which the new API returns. see 
https://github.com/exasol/error-reporting-python/blob/9a7ceb0b95b3b927be5d99c79b886d9ffa193be9/exasol/error/_error.py#L144

Reply:
And "Error" is not part of the module: `exasol.error`, see 
https://github.com/exasol/error-reporting-python/blob/9a7ceb0b95b3b927be5d99c79b886d9ffa193be9/exasol/error/__init__.py

Reply:
But you are using a private module, that is not good. We probably need to fix error reporting